libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/ManeuverMap.java

```diff
@@ -35,8 +35,12 @@
   ManeuverMap() {
     maneuverMap = new HashMap<>();
     maneuverMap.put(STEP_MANEUVER_TYPE_TURN + STEP_MANEUVER_MODIFIER_UTURN,
-      R.drawable.ic_maneuver_turn_180);
+      R.drawable.ic_maneuver_turn_180_left);
     maneuverMap.put(STEP_MANEUVER_TYPE_CONTINUE + STEP_MANEUVER_MODIFIER_UTURN,
+      R.drawable.ic_maneuver_turn_180_left);
+    maneuverMap.put(STEP_MANEUVER_TYPE_TURN + STEP_MANEUVER_MODIFIER_UTURN + STEP_MANEUVER_MODIFIER_LEFT,
```

Comment:
Just an idea, we could use bit-masking here instead of string concatenation so that the three qualifiers don't have to be in any particular order.

Reply:
Good point, let's revisit in a future PR. We should also consolidate `ManeuverMap`/ `ManeuverUtils` (notification) and `ManeuverViewMap` / `ManeuversStyleKit` (`ManeuverView`) code somehow. Refs. https://github.com/mapbox/mapbox-navigation-android/issues/1697